rs/nervous_system/canisters/src/ledger.rs

```diff
@@ -68,6 +78,18 @@ impl IcpLedger for IcpLedgerCanister {
         to: AccountIdentifier,
         memo: u64,
     ) -> Result<u64, NervousSystemError> {
+        tla_log_request!(
```

Comment:
I might structure this so that there's a TlaLoggingIcpLedgerCanister which wraps the IcpLedgerCanister - that could be defined in nns governance rather than here maybe?  I don't know that every canister that uses this IcpLedgerCanister is going to want to log the same events, right?  

The TLA logging should not spread across all the different crates, but should be contained inside of the nns governance canister as much as possible.

If it would be helpful, we could meet and talk through it.

I want the impact to the codebase to be narrowly defined.  If it ends up being tangled up everywhere, it's may add more maintenance than the benefits end up being worth.

Reply:
OK, let's discuss. A `TlaLoggingIcpLedgerCanister` may be possible, but I'd have to investigate (I don't know where/how the canisters are instantiated for the governance - you probably know off the top of your head). While we also have TLA models of the SNS (governance, swap) that could reuse the same logging if we want to expand the link to them later, I agree we want to keep the changes as local to the governance .

Reply:
Create `TlaLoggingIcpLedgerCanister` in governance and in `canisters.rs` instantiate either `IcpLedgerCanister` or `TlaLoggingIcpLedgerCanister` (e.g., create an alias and point it to one or the other depending on whether the feature is enabled).

Reply:
Created a tla_ledger.rs for the canister now.